caPutLogApp/caPutJsonLogTask.h

```diff
@@ -178,14 +183,24 @@ class epicsShareClass CaPutJsonLogTask {
      */
     std::map<std::string, std::string> getMetadata();
 
+    /**
+     * @brief Change the burst filter timeout
+     *
+     * @param timeout Time between puts to the same PV that will be considered a burst
+     * @return int Status code.
+     */
+    caPutJsonLogStatus setBurstTimeout(double timeout);
+
 private:
 
-    // Singleton instance of this class.
+    // Singelton instance of this class.
```

Comment:
This change introduced a spelling mistake that wasn't there before, please revert it: [Singleton pattern](https://en.wikipedia.org/wiki/Singleton_pattern)

Reply:
Fixed.